manifests/server/service.pp

```diff
@@ -10,10 +10,16 @@
     }
   }
 
+  if $mysql::server::override_options['mysqld'] and $mysql::server::override_options['mysqld']['user'] {
+    $mysqluser = $mysql::server::override_options['mysqld']['user']
+  } else {
+    $mysqluser = $options['mysqld']['user']
+  }
+
   file { $options['mysqld']['log-error']:
     ensure => present,
-    owner  => 'mysql',
-    group  => 'mysql',
+    owner  => $mysqluser,
+    group  => $::mysql::server::root_group,
```

Comment:
no problem, I introduced it, I should at least come up with a proposal to fix it. I kind of feel responsible for the sh*t I produce, ;) see #691